tools/distrib/buildozer.sh

```diff
@@ -34,12 +34,31 @@ function download_buildozer() {
         *)          error_handling "Unsupported platform: ${platform}";;
     esac
 
-    if [ -x "$(command -v curl)" ]; then
-        curl -L -o ${TEMP_BUILDOZER_PATH} ${download_link}
-    elif [ -x "$(command -v wget)" ]; then
-        wget -O ${TEMP_BUILDOZER_PATH} ${download_link}
-    else
-        error_handling "Download failed: curl and wget not available"
+    max_retry=5
```

Comment:
Nit: Recommend all variables/constants be `CAPS_SNAKECASE` and that all constant declarations be moved to the top.